tests/t0114_gateway_subdomains_test.go

```diff
@@ -1,13 +1,11 @@
-//go:build !no_subdomain_gateway_spec
-// +build !no_subdomain_gateway_spec
-
```

Comment:
Build tags are cool but I think they might not be powerful enough. E.g. if we had tests that are intersectional, then we'd have to create a new file for them. What I'm saying is, we could probably still achieve similar results but it'd be harder than if we used something that's available during runtime. The additional benefit is that with runtime exclusion, we can choose for the skipped tests to show up as skipped in the report. If we don't build a file, it's as if the tests never existed.